tests/endpoints/bookstore/bookstore_swagger_template.json

```diff
@@ -162,7 +162,11 @@
             }
           }
         },
-        "security": []
+        "security": [
+          {
+            "auth0_jwk": []
```

Comment:
Can you add security to a different endpoint? I believe `/shelves` is used by other tests as a basic test endpoint that requires no downstream security.

Reply:
I already add auth token to others requests so it should be fine.

I try to find one not depend on backend status(like bookId/shelfId ). Seems no other apis except this one and `get books`.

Reply:
Move to `GET /echo`

Reply:
Seems GET /echo is also used by load test. Change back to GET /shelves